--- origsrc/kwebkitpart/src/CMakeLists.txt	2017-07-16 14:02:26.000000000 -0500
+++ src/kwebkitpart/src/CMakeLists.txt	2017-07-16 14:06:23.796201100 -0500
@@ -22,7 +22,7 @@ qt5_wrap_ui(kwebkit_LIB_SRCS
 )
 
 ########################### kwebkitpart #############################
-add_library(kwebkitpart ${kwebkit_LIB_SRCS})
+add_library(kwebkitpart MODULE ${kwebkit_LIB_SRCS})
 
 target_link_libraries(kwebkitpart Qt5::Core Qt5::DBus Qt5::Gui Qt5::Widgets Qt5::WebKit Qt5::WebKitWidgets KF5::WebKit KF5::Parts KF5::KDELibs4Support KF5::SonnetCore)
 ########################### kwebkitpart #############################
