--- origsrc/kwebkitpart/cmake/modules/FindKWebKitPart.cmake	2010-04-28 19:04:28.000000000 -0500
+++ src/kwebkitpart/cmake/modules/FindKWebKitPart.cmake	2010-04-28 19:05:03.115359300 -0500
@@ -22,11 +22,11 @@ endif(kwebkitpart_SOURCE_DIR)
 
 # use pkg-config to get the directories and then use these values
 # in the FIND_PATH() and FIND_LIBRARY() calls
-if( NOT WIN32 )
+if( FALSE )
   find_package(PkgConfig)
   pkg_check_modules(PC_KWEBKITPART kwebkit)
   set(PCRE_DEFINITIONS ${PC_KWEBKITPART_CFLAGS_OTHER})
-endif( NOT WIN32 )
+endif( FALSE )
 
 FIND_PATH(KWEBKITPART_INCLUDE_DIR NAMES kwebkitpart.h
   PATHS
@@ -32,14 +32,14 @@ FIND_PATH(KWEBKITPART_INCLUDE_DIR NAMES
   PATHS
   ${PC_KWEBKITPART_INCLUDEDIR} 
   ${PC_KWEBKITPART_INCLUDE_DIRS}
-  ${KDE4_INCLUDE_INSTALL_DIR}
+  ${KDE4_INCLUDE_DIR}
 )
 
 FIND_LIBRARY(KWEBKITPART_LIBRARIES NAMES kwebkit
   PATHS
   ${PC_KWEBKITPART_LIBDIR} 
   ${PC_KWEBKITPART_LIBRARY_DIRS}
-  ${KDE4_LIB_INSTALL_DIR}
+  ${KDE4_LIB_DIR}
 )
 
 include(FindPackageHandleStandardArgs)
